### PR TITLE
remove unused code

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/cmds/cortex.py [hunk 1]
```diff
@@ -122,15 +105,24 @@ def save(self, mesg):
...
+        if fmt == 'jsonl':
+            s = json.dumps(mesg, sort_keys=True) + '\n'
+            buf = s.encode()
+            return buf
+
+        elif fmt == 'mpk':
+            buf = s_msgpack.en(mesg)
+            return buf
+
+        raise Exception('Unknown encoding format %s', fmt)
```
Comment (vEpiphyte) [inline synapse/cmds/cortex.py]: raise a SynErr

File: synapse/tests/test_lib_interval.py [hunk 2]
```diff
@@ -24,5 +24,5 @@ def test_ival_overlap(self):
         self.true(s_interval.overlap(ival2, ival3))
 
     def test_ival_parsetime(self):
-        ival = s_interval.parsetime('1970-1980')
+        s_interval.parsetime('1970-1980')
```
Comment (vEpiphyte) [inline synapse/tests/test_lib_interval.py]: why is this change here?  it doesn't actually mean anything.  was this a machine-change?

File: synapse/lib/cli.py [hunk 3]
```diff
@@ -222,26 +206,30 @@ def printf(self, mesg, addnl=True):
...
 
         if outp is None:
             outp = s_output.OutPut()
 
         self.outp = outp
         self.locs = locs
+
+        path = s_common.getSynPath('cmdr_history')
+
+        hist = FileHistory(path)
```
Comment (vEpiphyte) [inline synapse/lib/cli.py]: this is not okay since a unsafe test use of this object could muck up a real users cmdr_history

File: synapse/tools/pushfile.py [hunk 4]
```diff
@@ -49,7 +49,7 @@ def main(argv, outp=None):
         sha256 = fhashes.get('sha256')
         bsha256 = s_common.uhex(sha256)
 
-        awants = axon.wants([bsha256])
+        axon.wants([bsha256])
```
Comment (vEpiphyte) [inline synapse/tools/pushfile.py]: this is a nonsensical change. was it a machine made change?

File: synapse/lib/cli.py (hunk 3 above)
Comment (invisig0th) [reply synapse/lib/cli.py]: Good catch. Maybe we could/would/should make the test setup override something to redirect getSynPath() rather than having this problem everywhere?

Comment (vEpiphyte) [reply synapse/lib/cli.py]: previously we were lazily-loading the history when we were entering into the cmd loop; and only dealing with the actual file history *if* a local variable was set (which was set in the cmdr main()).  We could adopt that strategy again; but i don't know enough about prompt-toolkit ot know if we can delay the self.session initialization or not.

File: synapse/tests/test_lib_interval.py (hunk 2 above)
Comment (invisig0th) [reply synapse/tests/test_lib_interval.py]: This probably came from pyflake detecting that ival wasn't used.

Comment (invisig0th) [reply synapse/tests/test_lib_interval.py]: Yep.  Deleting the line, it's redundant with the line below.

File: synapse/cmds/cortex.py [hunk 5]
```diff
@@ -122,15 +105,25 @@ def save(self, mesg):
...
+            s = json.dumps(mesg, sort_keys=True) + '\n'
+            buf = s.encode()
+            return buf
+
+        elif fmt == 'mpk':
+            buf = s_msgpack.en(mesg)
+            return buf
+
+        mesg = f'Unknown encoding format: {fmt}'
+        raise SynErr(mesg=mesg)
```
Comment (vEpiphyte) [inline synapse/cmds/cortex.py]: `s_exc.SynErr`

File: synapse/lib/cli.py (hunk 3 above)
Comment (invisig0th) [reply synapse/lib/cli.py]: Moved session construction into the prompt() method.
